API-Mocks/WordPressMocks/src/main/assets/mocks/mappings/wpcom/notifications/notifications_comment_reply_after.json

```diff
@@ -0,0 +1,73 @@
+{
+	"scenarioName": "comment_flow",
+	"requiredScenarioState": "replied",
+	"newScenarioState": "new list",
+	"request": {
+		"method": "GET",
```

Comment:
the same here 🤔 the spacing looks off... but it's fine on my local:
![image](https://github.com/wordpress-mobile/WordPress-iOS/assets/17252150/ecd81249-3479-4e71-90f9-3d662f5985e4)

Reply:
Unlike the other mocks, this file has Tabs instead of 2 spaces indentation. We might want to fix that for consistency.